src/pages/MemberPage/AppointmentPlanCollectionBlock.tsx

```diff
@@ -21,7 +21,7 @@ const AppointmentPlanCollectionBlock: React.VFC<{ memberId: string }> = ({ membe
       <List>
         {enrolledAppointments.map(appointmentEnrollment => (
           <div key={appointmentEnrollment.orderProductId} className="mb-4">
-            <AppointmentCard {...appointmentEnrollment} onRefetch={refetchEnrolledAppointments} />
+            <AppointmentCard {...appointmentEnrollment} memberId={memberId} onRefetch={refetchEnrolledAppointments} />
```

Comment:
nitpick: I prefer using list props over spread operator.

Reply:
I agree, that using list props does indeed enhance readability. It has been adjusted.